codegen_glibmm/templates/common.cpp.templ

```diff
@@ -65,6 +65,11 @@ void Error::initialize()
     register_domain_with_dbus();
 }
 
+GQuark Error::registered_domain()
```

Comment:
Maybe it's better if we just return the Glib::Quark, and not a GQuark?

Reply:
Maybe, not sure. As mentioned in commit message, not even Glib::Error returns a Glib::Quark. And hum... should it be a Glib::QueryQuark instead? I do not know.

Thought it was less confusing to do same thing as Glib::Error::domain() does so it is possible to do as is done in the example in the commit message.

Reply:
FYI, one of the Glib::Error constructors also takes a GQuark and not a Glib::Quark.

Reply:
You are right: if Glib::Error::domain() returns a GQuark, then the best thing is to use GQuark here too.